### PR TITLE
remove unneeded localPlayer updateds

### LINKED ISSUE #2621: Performance tracker: track avatar CPU
Avatar CPU is not tracked but can easily be with a small refactor.

Right now vrm apps do not useFrame to update, but instead the update is triggered by the character controller, which is why the avatar CPU usage does not show up in the list. We need to move the player + avatar update handling to be under a useFrame call for the VRM app, which will get use avatar bones/physics CPU stats included in that app.

### PR DESCRIPTION
**Pull request by 0reo**
closes #2621 

this PR removes the update code which will not be needed once the vrm templates handle that via useFrame.  Currently requires https://github.com/webaverse/totum/pull/97 before it can be merged

### CODE REVIEW

**Comment by 0reo** [comment]
ready for review, https://github.com/webaverse/totum/pull/86 which this depends on is also waiting for review

File: webaverse.js [hunk 1]
```diff
@@ -292,19 +290,10 @@ export default class Webaverse extends EventTarget {
...
-          if (this.contentLoaded && physicsManager.getPhysicsEnabled()) {
-            physicsManager.simulatePhysics(timeDiffCapped);
-            localPlayer.updatePhysics(timestamp, timeDiffCapped);
-          }
 
           transformControls.update();
           game.update(timestamp, timeDiffCapped);
-          
-          localPlayer.updateAvatar(timestamp, timeDiffCapped);
-          playersManager.update(timestamp, timeDiffCapped);
```
Comment (avaer) [inline webaverse.js:306]: This line is removed and not added in the Torum PR. Where did this go?

**Review by avaer — CHANGES_REQUESTED** [review]
One question.

**Review by avaer — CHANGES_REQUESTED** [review]
No comments on the previous review, and the code has not changed.

https://github.com/webaverse/app/pull/2646#r834694958

**Comment by avaer** [comment]
The Totum PR has been merged but this one has not, so they are desynchronized and I suspect puling `totum` into master at this point would be broken.

**Comment by avaer** [comment]
Totum PR has been reverted: https://github.com/webaverse/totum/pull/86

So it will need to be submitted and merged synchronously with this one next time.

**Comment by 0reo** [comment]
new totum PR with useFrame was added to this.  have not recieved any feedback regarding the review notes, but both this and totum should be ready to merge.

**Comment by 0reo** [comment]
closing this PR as there will be a general change in approach.  will be addressed in a new PR

**Pull request closed without merging**